### ISSUE TITLE
Did not manage to remap AltGr

### ISSUE THREAD

**Issue by jloow** [randyrants/sharpkeys#202]
I tried to remap AltGr to Left Alt but with no success.

From what I can tell, AltGr becomes disabled after remapping it. After rebooting, AltGr appears as Right Alt in SharpKeys. Rebinding Right Alt to Left Alt does not work as a substitute, however.

**Comment by randyrants** [comment]
Can you share a screenshot of the mapped keys?  I don't have access to a keyboard with AltGr but I also haven't heard of this issue from anyone else. (yet).

**Comment by caxml** [comment]
I successfully remapped AltGr (E0_2038) to Left Alt (00_38) in version 3.7.0. 
After logging out and back in, what was earlier AltGr now shows as Right Alt (E0_38), but AltGr still functions as Left Alt.

Updated to 3.9.0, same results.

This is what it looks like after relogging, AltGr is mapped to LeftAlt.
![sharpkeys](https://user-images.githubusercontent.com/26446541/66020140-60d9ac80-e4e6-11e9-9e8d-b4859aa21f9e.JPG)
